Makefile

```diff
@@ -17,6 +18,7 @@ CFLAGS+=-std=gnu89 -O2 -g -MMD -Wall				\
 	-DNO_BCACHEFS_FS					\
 	-DNO_BCACHEFS_SYSFS					\
 	-DVERSION_STRING='"$(VERSION)"'				\
+	-Iinclude						\
```

Comment:
Why moving this around?

Reply:
see first coment: I'm experimenting with options to build against kernel headers.  This one is a leftover which shouldn't have been there